cf_xarray/accessor.py

```diff
@@ -65,6 +65,16 @@
 ATTRS["time"] = ATTRS["T"]
 ATTRS["vertical"] = ATTRS["Z"]
 
+OPTIONS: MutableMapping[str, Any] = {"custom_criteria": []}
+
+
+def set_options(custom_criteria):
+    OPTIONS["custom_criteria"] = (
+        always_iterable(custom_criteria, allowed=(tuple, list, set))
+        + OPTIONS["custom_criteria"]
+    )
```

Comment:
I think it would fail with if custom criteria is a tuple or a set.
```suggestion
def set_options(custom_criteria):
    OPTIONS["custom_criteria"].extend(
        always_iterable(custom_criteria, allowed=(tuple, list, set))
    )
```